src/components/Layout/LayoutStyled.js

```diff
@@ -1,24 +1,25 @@
 import styled from 'styled-components';
 
-// export const Container = styled.div`
-//   display: flex;
-//   width: 100%;
-//   min-height: 100vh;
-//   margin: 0 auto;
-
-//   @media screen and (min-width: 375px) {
-//     max-width: 375px;
-//   }
-//   @media screen and (min-width: 768px) {
-//     max-width: 1440px;
-//   }
-//   @media screen and (min-width: 1440px) {
-//     max-width: 1920px;
-//   }
-// `;
-
 export const LayoutStyled = styled.div`
   display: grid;
   grid-template-columns: 120px 1fr;
   background-color: var(--outlet-background-color);
 `;
+
+export const LogoBlock = styled.div`
```

Comment:
I don't see any places you use these two components. Do we really need it?

Reply:
Yes it will be first loading of Chat if the user hasn't any opened chat room yet. But I can delete it now and add it later